lib/common/grpc-service.js

```diff
@@ -457,4 +465,37 @@ GrpcService.prototype.getGrpcCredentials_ = function(callback) {
   });
 };
 
+/**
+ * @private
+ */
+GrpcService.prototype.loadProtoFile_ = function(protoConfig, config) {
```

Comment:
Previously we would load our <kbd>.proto</kbd> files via `for .. in` loop, however since the complexity for programmatically loading these files has been increased, I decided to move that logic to its own method. 

This is setup to allow for *overrides* in the event of conflicting namespaces/versions/etc. between files.